src/tests/test_magic_cmd.py

```diff
@@ -125,6 +125,54 @@ def test_error(tmp_empty, ip, cell, error_message):
     assert str(excinfo.value) == error_message
 
 
+@pytest.mark.parametrize(
+    "command",
+    [
+        "tables",
+        "columns",
+        "test",
+        "profile",
+        "explore",
+    ],
+)
+def test_sqlcmd_error_when_no_connection(ip_empty, command):
+    with pytest.raises(UsageError) as excinfo:
+        ip_empty.run_cell(f"%sqlcmd {command}")
+
+    assert excinfo.value.error_type == "RuntimeError"
+    assert str(excinfo.value) == "No active connection"
+
+
+def test_sqlcmd_snippets_when_no_connection(ip_empty, capsys):
+    for key in list(store):
+        del store[key]
```

Comment:
I added a fixture that cleans the snippet store before a tests is execute, try removing this, it should work

https://github.com/ploomber/jupysql/blob/02ba21cf47b5df6c2191bf5d60823eae0ce0b7e4/src/tests/conftest.py#L24

Reply:
It seems `isolate_tests` isn't working. As shown [here](https://github.com/ploomber/jupysql/actions/runs/5895363136/job/15991001391?pr=804#step:6:63), there are snippets from `test_magic` inside the store when I tested `test_sqlcmd_snippets_when_no_connection` without using the `del store[key]`.

When I ran tests locally, the tests work when I just run `test_magic_cmd.py`. So it seems like the stored snippets got carried over from `test_magic.py` to `test_magic_cmd.py`.

And I tried testing `test_sqlcmd_snippets_when_no_connection` after a test that uses `ip_snippets` and confirmed that `store` from `test_sqlcmd_snippets_when_no_connection` contains snippets from the `ip_snippets` fixture.

Reply:
ok, put back the for loop that you're using to clean up the store so your tests pass and open an issue so I can investigate. let me know when this is ready for review!

Reply:
[revert](https://github.com/ploomber/jupysql/pull/804/files#diff-9188e75e824219dd191cdc62d6d19f2b15c4bd0cf564abe3646ea907c32cd61dR150-R151)